napalm/ios/ios.py

```diff
@@ -2110,7 +2110,7 @@ def get_environment(self):
         re_temp_value = re.compile("(.*) Temperature Value")
         # The 'show env temperature status' is not ubiquitous in Cisco IOS
         output = self._send_command(temp_cmd)
-        if "% Invalid" not in output:
+        if "% Invalid" and "Not Supported" not in output:
```

Comment:
I think `if "% Invalid" not in output or "Not Supported" not in output:` might be what you want.

Reply:
@mirceaulinic You are 'more right' than me, but still not entirely correct: It's `if "% Invalid" not in output and "Not Supported" not in output:` - `and` instead of `or` :) I will try to update my PR, not sure how to do that...